code/app/src/main/res/layout/entrant_home_view.xml

```diff
@@ -5,78 +5,91 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <!-- Profile Image -->
     <ImageView
-        android:id="@+id/entrant_home_view_image_view"
+        android:id="@+id/profileImage"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="32dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:srcCompat="@tools:sample/avatars" />
+        tools:srcCompat="@tools:sample/avatars[5]" />
 
+    <!-- Username Title Text -->
     <TextView
-        android:id="@+id/entrant_home_view_user_name"
+        android:id="@+id/username"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="16dp"
-        android:text="Name"
+        android:text="Username Placeholder"
+        android:textSize="18sp"
+        android:textStyle="bold"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/entrant_home_view_image_view" />
```

Comment:
please leave this id unchanged as well as it is more descriptive and easier to find

Reply:
this one should also be done now as well